src/meshdb/utils/spreadsheet_import/parse_link.py

```diff
@@ -121,12 +121,7 @@ def get_representative_device_for_node(node: Node, link_status: SpreadsheetLinkS
         device = Device(
             node=node,
             name=f"NN{node.network_number} Core",
-            model="Unknown",
-            type=Device.DeviceType.OTHER,
             status=status,
-            latitude=node.latitude,
-            longitude=node.longitude,
-            altitude=node.altitude,
```

Comment:
These were removed from `Device` but i missed this instance of the constructor invocation